packages/salto-analyzer/salto-analyzer.0.1/opam

```diff
@@ -60,4 +60,4 @@ url {
     "sha512=6efc54d9ff97fc5654c28a6c8f96d814be44cf53e03d301c907b5b2178813a0f7837e341272891b92417cc98529acc8a66062002e16e705c781a3ec2a3db5f16"
   ]
 }
-x-commit-hash: "052c78a048375988dc8649cbeb45d156f031a732"
+x-commit-hash: "f43c5de88307bf1a360b40be9e995ae3e63bb1ad"
```

Comment:
Why is this changed?

Reply:
That's something I forgot to revert. This is now fixed with the latest push.